### PR TITLE
Fix `zos add` in Windows environments

### PR DESCRIPTION
**Pull request by siromivel**
Addresses #759 

This was a very quick fix and I have both Windows and Linux environments available at home. I've tested it in my Linux environment and everything seems fine; I'll try running the tests in my Windows environment tomorrow when I get home.

### CODE REVIEW

**Comment by siromivel** [comment]
Ah unfortunately I don't have access to a Mac machine for testing at the moment; if there is an issue there it looks like it should be possible to check if the OS is windows-based and conditionally run an `exec` in that case although it would be nicer to not need switching logic.

**Comment by spalladino** [comment]
Actually the issue with having spaces in the path affected both Linux and MacOS. The quick and dirty solution was to quote the executable, but using `execFile` instead was a cleaner solution. That said, I didn't expect it to break Windows, though it seems that the differences are [platform-dependent indeed](https://nodejs.org/api/child_process.html#child_process_spawning_bat_and_cmd_files_on_windows).

@siromivel can I ask you to test using `execFile` with the `shell` flag set to true, and see if it works in Windows? I don't have a Windows dev station setup here to test it out now. Thanks!

**Comment by siromivel** [comment]
Just an update: I'm working on testing this on my Windows machine, running into some issues as the current User directory has spaces and is causing different issues 😂

**Comment by siromivel** [comment]
Ok I've confirmed that simply passing in the `shell: true` arg fixes this issue on a Windows machine and doesn't appear to break Linux compatibility. I can't test this on MacOS currently.

image
🍾

File: packages/cli/src/models/compiler/Compiler.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { execFile, ExecException } from 'child_process';
-import { FileSystem, Contracts, Logger } from 'zos-lib';
+import { FileSystem, Logger } from 'zos-lib';
```
Comment (siromivel) [inline packages/cli/src/models/compiler/Compiler.ts]: This was an unused import

**Comment by siromivel** [comment]
I'm seeing a build issue on Travis with this change, I'll look into that a bit later this morning.

**Comment by siromivel** [comment]
Build is fixed - I was passing the args as an object literal which TypeScript didn't like, not sure why it was compiling locally but moving those out into a const assignment got the build working again.

**Pull request merged**